code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,41 @@
+import request from 'supertest'
+import express from 'express'
+import graphqlHTTP from 'express-graphql'
+import schema from '../../setup/schema'
+
+describe('user mutations', () => {
+  let server;
+
+  beforeAll(() => {
+    server = express();
+    server.use('/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false
+      })
+    );
+  })
+
+  it('changes emails', async () => {
+    const response = await request(server)
+      .post('/')
+      .send({ query:
+        'mutation { userUpdate(id: 2, email: "user@email.com") { email } }'
```

Comment:
So it was the ":" that was breaking it before?!

Reply:
Sorta - you have to send it in as a 'query' block, regardless of whether it's a mutation or just a query. If it is a mutation though, you have to write out the code like you'd send it into GraphQL, instead of it implicitly understanding that this is a 'query'.